fedoratagger/lib/retired.py

```diff
@@ -23,49 +23,55 @@
 log.setLevel(logging.DEBUG)
 logging.basicConfig()
 
-PKGDB_URL = 'https://admin.fedoraproject.org/pkgdb/api/packages'
+PDC_URL = 'https://pdc.fedoraproject.org/rest_api/v1/component-branches'
 
-def get_packages(status):
-    log.info('Retrieve the list of %s packages.' % status)
-    total = 1
+
+def get_packages(is_active=False):
+    log.info('Retrieve the list of packages.')
     page = 1
+    next_page = 1
     packages = []
-    while page <= total:
+    while next_page:
         args = {
-            'status': status,
+            'active': is_active,
             'page': page,
         }
-        r = requests.get(PKGDB_URL, params=args)
+        log.info('Querying page %s' % page)
+        r = requests.get(PDC_URL, params=args)
         output = json.loads(r.text)
-        if hasattr(output, 'error'):
-            log.error(output['error'])
-            return packages
+        for pkg in output['results']:
+            pkg_name = pkg['name']
+            log.info('Adding %s to packages' % pkg_name)
+            packages.append(pkg_name)
 
-        total = output['page_total']
-        for pkg in output['packages']:
-            packages.append(pkg['name'])
+        next_page = output['next']
```

Comment:
`output['next']` gives you the next page url, so to get the next page you need to call this url instead of the `PDC_URL`.